src/test/java/org/broadinstitute/hellbender/tools/walkers/ReferenceConfidenceVariantContextMergerUnitTest.java

```diff
@@ -205,6 +211,13 @@ public Object[][] makeReferenceConfidenceMergeData() {
                         new GenotypeBuilder("A_C_G.test2").PL(new int[]{40, 20, 30, 20, 10, 30}).alleles(noCalls).make(),
                         new GenotypeBuilder("A_C_G.test").PL(new int[]{40, 20, 30, 20, 10, 30}).alleles(noCalls).make()).make()});
 
+        // test creation of AD with proper allele indexing without PLs
+        // this needs to be multi-allelic
+        tests.add(new Object[]{"test12",Arrays.asList(vcA_C_G_noPLs, vcA_C_G_ALT_noPLs), loc, false, true,
+                new VariantContextBuilder(VCbase).alleles(A_C_G).genotypes(
+                        new GenotypeBuilder("A_C_G.test2").AD(new int[]{60,9,0}).alleles(noCalls).make(),
+                        new GenotypeBuilder("A_C_G.test").AD(new int[]{60,9,0}).alleles(noCalls).make()).make()});
```

Comment:
The variant context has Arrays.asList(Aref, C, G, Allele.NON_REF_ALLELE) so the ADs should be of length 4.  Do you mean LAD?  If this is how you want it for GVS, then I'd like to make that more obvious some how.

Reply:
hmmmm then I'm not testing what I think I'm testing....

Reply:
in the test (on line 51) we hardcode the filtering out of the Allele.NON_REF_ALLELE
(  @param removeNonRefSymbolicAllele if true, remove the <NON_REF> allele from the merged VC ) so it's not local AD and should not have that additional AD value.